libs/components/src/avatar/avatar.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { Meta, moduleMetadata, Story } from "@storybook/angular";
+
+import { CryptoFunctionService } from "@bitwarden/common/abstractions/cryptoFunction.service";
+import { StateService } from "@bitwarden/common/abstractions/state.service";
+
+import { AvatarComponent } from "./avatar.component";
+
+export default {
+  title: "Component Library/Avatar",
+  component: AvatarComponent,
+  decorators: [
+    moduleMetadata({
+      providers: [
+        CryptoFunctionService,
+        {
+          provide: StateService,
+          useValue: {
+            getEnableGravitars: () => {
+              return;
+            },
+          },
+        },
+      ],
+    }),
+  ],
+  args: {
+    name: "Jason Doe",
+  },
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/Zt3YSeb6E6lebAffrNLa0h/Tailwind-Component-Library?node-id=1717%3A15868",
```

Comment:
This url doesn't seem to go to the avatar component.

```suggestion
      url: "https://www.figma.com/file/Zt3YSeb6E6lebAffrNLa0h/Tailwind-Component-Library?node-id=1881%3A16994",
```